pom.xml

```diff
@@ -52,12 +52,13 @@ agnostic home for software distribution comprehension and audit tools.
     <maven.compiler.source>${javaVersion}</maven.compiler.source>
     <maven.compiler.target>${javaVersion}</maven.compiler.target>
     <!-- This is the version of Maven required to use the Rat Maven Plugin -->
-    <mavenVersion>3.2.5</mavenVersion>
+    <mavenMinVersion>3.2.5</mavenMinVersion>
+    <mavenVersion>3.9.6</mavenVersion>
```

Comment:
fyi: I started a discussion on the mailing list concerning this update ...... let's see if there are any complaints about that one.

Reply:
The point of this update: "we" (Maven project) did it wrong, compile against latest (3.9.6) but allow runtime 3.2.5 (min maven). The reason to do so is to pick up deprecations and get notified what is changing, but source and binary compatibility guarantees it to run on min version as well. If you compile against 3.2.5 forever, you get totally disconnected about latest changes and deprecations in Maven APIs.

Reply:
Also, this affects _compilation only_, as at runtime, whatever Maven version you use, will provide the runtime stuff, and this does not mean Maven 3.9.6 is must or needed or whatever. Again, this is only to get compiler emit deprecation messages to notify devs (like use of deprecated type ArtifactRepository in one mojo is).

Reply:
Thanks for this context. So basically it's the same as https://github.com/apache/creadur-rat/pull/172 concerning the maven deps?

Reply:
Not quite: that is "machine" going for highest, for example it left unreal "min" version in place:
https://github.com/apache/creadur-rat/blob/b0442fd49026b1929e9a88f2ea1c655dca106115/pom.xml#L469

Maven 3.0.x should be avoided (due package rename happened between 3.0.x and 3.1.x). We in Maven also set "lowest version" to 3.2.5 these days, it covers from 2014 until today, I hope 10 years is good enough.